index.js

```diff
@@ -53,6 +53,26 @@ module.exports = function postcssPrefixSelector(options) {
   };
 };
 
+function ignoreFile(file, ignoreArr) {
+  return ignoreArr.some((ignoreRule) => {
+    if (ignoreRule instanceof RegExp) {
+      return ignoreRule.test(file);
+    }
+
+    return file.includes(ignoreRule);
+  });
+}
+
+function includeFile(file, includeArr) {
```

Comment:
Same as above, please make the name more descriptive, like `isFileIncluded`

Reply:
Also, both these functions do basically the same thing. Can you merge them into one and give it a descriptive name?